### PR TITLE
Always run UI tests for backend changes.

### PR DESCRIPTION
**Pull request by khdegraaf**
To verify if any backend changes break UI.

### CODE REVIEW

File: .github/workflows/node.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Node.js CI
 
 on:
   pull_request:
-    paths: ['ui/**']
+    paths: ['ui/**', '**.go']
```
Comment (banshee) [inline .github/workflows/node.yml:5]: The title of the PR says always run for backend changes, but this says just run very specifically for .go files.  What's the goal?

Comment (khdegraaf) [reply .github/workflows/node.yml:5]: Our backend changes are triggered by **.go.  So this is the same.  If this is missing important stuff, so is our backend trigger.  Do you have an instance where you think this is wrong?

Comment (banshee) [reply .github/workflows/node.yml:5]: Changes to go.mod?

Comment (banshee) [reply .github/workflows/node.yml:5]: Or maybe the Makefile that builds the go stuff could have a subtle change in the way things are built?

Comment (banshee) [reply .github/workflows/node.yml:5]: The action ends up calling 

```run: yarn chain:build && yarn build```

So it builds and uses just about everything.  We can't limit the file set without a lot of work and maintenance.

Comment (khdegraaf) [reply .github/workflows/node.yml]: I removed the paths dependencies so we always get this.

**Review by banshee — APPROVED** [review]

**Review by ryardley — APPROVED** [review]
🚀

**Pull request merged**